src/js/helpers/monthHelper.js

```diff
@@ -39,3 +39,21 @@ export const convertPeriodToDate = (period, fy) => {
     const year = (parsedPeriod <= 3) ? parsedFy - 1 : parsedFy;
     return `${month} ${year}`;
 };
+
+export const fullMonthFromAbbr = (abbr) => {
```

Comment:
this should just be a dataMapping object. We might also already be doing this elsewhere.

```javascript
  const monthByMonthAbbreviation = {
      'Jan': 'January'
      // etc...
  }
  const month = monthByMonthAbbreviation[abbr];
```

Reply:
I searched the code, didn't see this specific conversion in this project. Maybe in an imported one...